msr-flows/Notes/type_1_flow.ipynb

```diff
@@ -0,0 +1,152 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Linear Growth Approximation\n",
+    "\n",
+    "#### Only suitable for SS\n",
+    "\n",
+    "The linear growth approximation is an approach which assumes that over some time step X, the growth is approximately linear. This means that the removal constant must be calculated such that the number of atoms at time X is the same as SaltProc at that time. As long as accuracy can be maintained to some position past where the SaltProc value is determined, then this method can be used to reduce the computational time of SaltProc by making a series of small linear approximations.\n",
+    "\n",
+    "![title](LGA.png)\n",
+    "\n",
+    "This approach has 3 main steps\n",
+    "\n",
+    "1. Run Serpent with no reprocessing for X days\n",
+    "\n",
+    "    1a. This gives enough data to create an approximate linear growth term\n",
+    "    \n",
+    "    $$\\frac{dN}{dt} = C$$\n",
+    "\n",
+    "    $$C = \\frac{N_f - N_i}{t_f - t_i}$$\n",
+    "\n",
+    "2. Plug the constant growth value in and then include reprocessing with the condition that it must match SaltProc at a given time. This is done by calculating waste atoms and solving with scipy.optimize root function.     \n",
```

Comment:
Flow + formatting edit for enhanced clarity. Same quantifier on your intended message applies to this suggestion.
```suggestion
    "2. Plug the linear growth term into the Bateman equation and then include reprocessing with the condition that it must match the SaltProc value at a given time. This is done by calculating waste atoms and solving with `scipy.optimize` root function.     \n",
```

Reply:
tagging @nsryan2 for another opinion